docs/content/en/cookbook/store.md

```diff
@@ -121,14 +121,14 @@ See [this issue](https://github.com/michaelolof/vuex-class-component/issues/43)
 Vuex supplies very basic types for use with the store. You can use these to help define your store. For example:
 
 ```ts{}[store/index.ts]
-import { GetterTree, ActionTree, MutationTree } from 'vuex'
+import { GetterTree, ActionTree, MutationTree, Store } from 'vuex'
 
 export const state = () => ({
   things: [] as string[],
   name: 'Me',
 })
 
-export type RootState = ReturnType<typeof state>
+export type RootState = Store<ReturnType<typeof state>>
```

Comment:
This is not correct. State is NOT an instance of `Store` and it would for example break the getter below. You can try pasting this code into a typescript file and see.

Reply:
So then how should I get $axios? in fact it is there.

Reply:
Typically by adding the module to `types` in `tsconfig.json`. In this case I guess `@nuxtjs/axios`.

Reply:
But it will not pass ензу ща $axios any way.
But in reality we can access this variable.
I spend few days to figure it out.
So thats why I wanna share my solution.

Reply:
It's not a correct solution though. It breaks other things.
Maybe your actions are not typed correctly. Hard to say without seeing the actual code.
If you prepare a minimal repo that reproduces then maybe someone can help you.

Reply:
I tried to figure out why I did it, but I couldn't.
Much time has passed since then.
Have a nice day.